core/services/blockhashstore/feeder.go

```diff
@@ -114,3 +132,114 @@ func (f *Feeder) Run(ctx context.Context) error {
 	f.lastRunBlock = latestBlock
 	return errs
 }
+
+func (f *Feeder) runTrusted(
```

Comment:
So the trusted BHS will only ever run trusted stores - we'll still run a regular BHS job to do the non-trusted stuff right?

Reply:
Just thinking about this, but do we need to update the BHS job to do trusted stores? Or should we just have a script to do the trusted store (similar to the backwards mode) in extraordinary situations?

Reply:
We can definitely add a script, but this logic is probably more robust than the regular BHS behavior since it can have a much greater lookback and fulfills in batches. Can probably keep it now that it exists, but happy to consider otherwise.